src/develop/ui/patterns/mobile/columns.md

```diff
@@ -3,26 +3,31 @@ tags: runtime-mobileandreactiveweb;
 summary: 
 ---
 
-# Columns Pattern
+# Columns 
 
-The Columns pattern 
-enables you to split content into multiple columns. You can configure the
-behavior using the input parameters. They will define how columns will stack
-in different devices. This is ideal to improve the way information is
-displayed across different devices. You can use this pattern to display a list
-of elements side by side, with a different number of items per row on
-different devices.
+You can use the Columns UI Pattern to split content into multiple columns. Additionally, you can configure the column behavior and define how they display on different devices. Splitting content into columns improves how information is displayed and enhances the overall user experience.
 
-Here's the preview , in Service Studio, of the different ways columns can be split into:
+![](images/columns-image-4.png)  
+
+<!---You can split the columns as follows:
 
 ![](images/Column_Gutter.png)
 
-![](images/Column_columns.png)
+![](images/Column_columns.png)--->
+
+## How to use the Columns UI Pattern
+
+1. In Service Studio, in the Toolbox, search for `Columns`.
+  
+     The various Column widgets are displayed.
+
+    ![](images/columns-image-6.png)  
+
+1. From the Toolbox, drag the relevant Columns widget onto your application's screen. 
 
-## How to Use the Columns Pattern
+1. On the **Properties** tab, configure the column behavior to define how the columns will display on different devices.
 
-Configure the behavior using the input parameters to define how columns will
-stack in different devices.
+After following these steps and publishing the module, you can test the pattern in your app.
 
 For an uneven number of columns, the following splits apply:
```

Comment:
These examples should perhaps be after the Properties table, and under a section called "Column Breaks examples" or similar? @pacferreira